test/js/bun/util/inspect.test.js

```diff
@@ -520,3 +520,24 @@ it("Bun.inspect array with non-indexed properties", () => {
   1, 2, 3, 15 x empty items, 24, 23 x empty items, potato: "hello"
 ]`);
 });
+
+// it("console.logging function displays async and generator names", async () => {
```

Comment:
This code can be deleted
```suggestion
// it("console.logging function displays async and generator names", async () => {
```